<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAccounts</class>
 <widget class="QDialog" name="MyAccounts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Accounts</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Account Number : </string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="balance">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="accountNum">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="CreditCharge">
     <property name="text">
      <string>Credit Charge</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QPushButton" name="bill">
     <property name="text">
      <string>Paying Bill</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="loan">
     <property name="text">
      <string>Loan</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="requestCard">
     <property name="text">
      <string>Request Card</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="Cheque">
     <property name="text">
      <string>Cheque</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Balance :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="accountType">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Account Type :</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="buyingNet">
     <property name="text">
      <string>Buying Internet</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="transferByAccount">
     <property name="text">
      <string>Transfer Money By Account</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="next">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="befor">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
